modules/serverless-config-replicas-eventing.adoc

```diff
@@ -45,3 +45,43 @@ spec:
   high-availability:
     replicas: 3
 ----
+
+. You can also specify the number of replicas for a specific workload:
```

Comment:
Probably nix the `:` since it's immediately followed by a note block
```suggestion
. You can also specify the number of replicas for a specific workload.
```